python/lsst/meas/astrom/astrometry.py

```diff
@@ -60,6 +60,15 @@ class AstrometryConfig(RefMatchConfig):
         default=0.001,
         min=0,
     )
+    maxMeanDistanceArcsec = pexConfig.RangeField(
+        doc="Maximum mean on-sky distance (in arcsec) of a WCS fit beyond which the fit is "
+            "considered a failure.  Default is set to maximum tolerated by the external global "
```

Comment:
I think this should say what exception is raised if it fails (`TaskError` I guess).

Reply:
Reading this again, I think that "distance of a WCS fit" doesn't quite make sense.  Perhaps something like "Maximum mean on-sky distance (in arcsec) between the sources and matched reference stars post-fit.  Mean distance greater than this threshold raises a TaskError and the WCS fit is considered a failure.  Default is set ..."